project/Build.scala

```diff
@@ -149,9 +149,9 @@ object Build {
   val mimaPreviousLTSDottyVersion = "3.3.0"
 
   /** Version of Scala CLI to download */
-  val scalaCliLauncherVersion = "1.9.1"
+  val scalaCliLauncherVersion = "1.10.0"
   /** Version of Coursier to download for initializing the local maven repo of Scala command */
-  val coursierJarVersion = "2.1.24"
+  val coursierJarVersion = "2.1.25-M19"
```

Comment:
Would prefer to use stable versions, but I understood the decision. We're getting only M releases for the last 9 months...

Reply:
Unfortunately, yeah.